lib/web/mage/validation.js

```diff
@@ -194,6 +194,7 @@
         var empty = $(element).closest('table')
             .find('input.required-option:visible')
             .filter(function (i, el) {
+                if($(element).is(":visible"))
```

Comment:
Why do we need to check visibility of the element while we are iterating the child elements?

Reply:
Because in attributes like country of manufacture, the attribute "required-dropdown-attribute-entry" (not visible) is failing because the default value of the country list is empty.

Reply:
"required-dropdown-attribute-entry" this is attribute on element that will be hidden always, it's a "marker" for errors. This element was introduced to decrease validation time. Before commit, that you mentioned in description, validation process collected each field in each row in result it was many objects that validates separately. So, you must check for `$(el).is(':disabled')`, because your fix is incorrect and it's make full skip of validation on required label.